context/config.ts

```diff
@@ -10,3 +10,5 @@ export const PREFERED_NETWORK =
   process.env.NEXT_PUBLIC_NETWORK_TYPE === "mainnet"
     ? NetworkType.MAINNET
     : NetworkType.GHOSTNET;
+
+export const DEFAULT_TIMEOUT = 90000;
```

Comment:
currently, one block/conformation is about 30s. I think we should be shorter, maybe 45s?

Reply:
Oh didn't know, you're right sounds a bit too long. But I would probably go with 60s to make sure even slow connections have the time to get a confirmation, wdyt?

Reply:
sounds good.